### DISCUSSION TITLE
Welcome to ReEDS_OpenAccess Discussions!

### DISCUSSION THREAD

**Discussion by keurek** [NREL/ReEDS-2.0#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aucci** [comment]
Hello,
I'm new to ReEDS and hope to use it to study renewable energy/decarbononization in the New England region.  At this point, I just got ReEDS up and running so looking forward to learning more.  Not sure what or how much people usually share in this forum.

**Comment by maxxb77** [comment]
We love to hear how you’re using ReEDS and very much like to learn how we
could make it better. I’ve modeled quite a few policy setups over the
past few years - would be cool to hear how you’re approaching the
representation.
